src/app/components/sidenav/sidenav.component.scss

```diff
@@ -0,0 +1,89 @@
+.main-sidebar {
+  display: flex;
+  flex-direction: column;
+  width: 280px;
+  transition: width 0.08s;
+  margin-top: 1em;
+  @media (max-width: 599px) {
```

Comment:
do we need media queries in our project please check the project and remove media queries if we can